packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -259,13 +286,18 @@ governing permissions and limitations under the License.
        grid-template-rows: calc(var(--spectrum-dialog-padding) + var(--spectrum-dialog-fullscreen-padding-top)) auto auto 1fr auto var(--spectrum-dialog-padding);
        grid-template-areas:
          ".    .       .            ."
-         ".    header  header       ."
+         ".    heading header       ."
          ".    divider divider      ."
          ".    content content      ."
          ".    .       buttonGroup  ."
          ".    .       .            .";
     }
 
+    .spectrum-Dialog-heading {
```

Comment:
See conversation in slack. Spectrum design decided to make this the style for all fullscreen dialogs, regardless of screen size.